### DISCUSSION TITLE
How to let Apps access /mnt folder and network drives?

### DISCUSSION THREAD

**Discussion by silasrogan** [IceWhaleTech/CasaOS#842] [Q&A]
/mnt drives aren't visible, though I can pass a network drive if I map it in the Docker compose for the particular app. Is this something that's possible? Do I need to change permissions (and if so, would I do that through Ubuntu or CasaOS)?

I'm trying to have an instance of sabNZBD to test against the one on my NAS, but I have no access even after I "map" the drive.

Thanks!

**Comment by CosmoFox** [comment]
It`s easy. Just use symlink.
As example:
1. mkdir in DATA folder (sudo mkdir /Data/mnt)
2. make symlink (sudo ln -s /mnt/$drive /Data/mnt)
3. be happy

Or shorter (entire /mnt folder):
  1. make symlink (sudo ln -s /mnt/ /Data)
  2. be happy

**Reply by stilzkyn** [reply]
I tried that option and still nothing.
